index.js

```diff
@@ -0,0 +1,31 @@
+var calculator = new Vue({
+    el: '#calculator',
+    data: {
+        price: '',
+        tipRate: '20',
+        total: ''
+    },
+    computed: {
+        numTipTotal: function(e) {
+          var percentage = this.tipRate/100 + 1;
+          var calculatedTotal = percentage * this.price
+          return calculatedTotal
+        },
+        calcTotal: function(e){
+            return currencyFormat(this.numTipTotal)
+        },
+        numFormat: function(e){
+            e.replace(/(\d)(?=(\d\d\d)+(?!\d))/g, "$1,");
```

Comment:
I"ve never looked at vue before but I don't really understand what this line does? Maybe more a quick comment to explain?
Never reviewed code before so please bare with me!
App overall looks great!!